### ISSUE TITLE
Possible customizer issue with a Google Analytics plugin

### ISSUE THREAD

**Issue by ghost** [AlxMedia/hueman#685]
> I just today updated Hueman and Add-Ons. It disabled my homepage sidebar layout. On trying to open the Customizer it would not open and the only way I was able to get it to function was by disabling Google Analytics plugin. It then forced me to utilise the Social Links in order to be able to save settings in the Customizer.
https://wordpress.org/support/topic/latest-update-42/#post-10446106

**Comment by ghost** [comment]
additional informations provided by the user

> Hi Nick I disabled both of these to get Customizer back – Google Analyticator, Google Analyitrics for WordPress by MonsterInsights.
> 
> Also this morning got this warning:
> 
> MAIL DELIVERY ERROR: WP Mail SMTP plugin logged this error during the last time it tried to send an email:
> 
> Mailer: Default (none)
> PHPMailer was able to connect to SMTP server but failed while trying to send an email.
> Please review your WP Mail SMTP settings in plugin admin area. Consider running an email test after fixing it.

**Comment by eri-trabiccolo** [comment]
Ok I've tested Google Analyticator, Google Analyitics for WordPress by MonsterInsights + Hueman Addons + Hueman: no issue.

I've then installed https://wordpress.org/plugins/wp-mail-smtp/
And when accessing the customizer I get this:
<img width="957" alt="2018-06-29_10-32-30" src="https://user-images.githubusercontent.com/7689242/42082236-c266198a-7b87-11e8-94f1-1fdde19e0068.png">

If this is the issue the user is referring to, then not Hueman nor the Hueman Addons has anything to do with it: it occurs with TwentySeventeen too (and Hueman Addons disabled) as long as WP Mail SMTP + Google Analyticator plugins are active. Disabling one of them will make the customizer work as it should.

The fact that the "issue" occurred after the Hueman+Addons update is just due to a coincidence IMHO, probably at the same time the user updated those other plugins too, and/or he didn't access the customizer ever since he installing them and noticed it just now.

About the PHPMailer warning: cannot really reproduce any issue, but I doubt it's related to the theme+addon.

I would set this issue as invalid then.
